minecraft_servers/operamc/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "server_name": "operamc",
+  "nice_name": "OperaMC",
+  "direct_ip": "operamc.de",
+  "server_wildcards": [
+    "%.operamc.de"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "tiktok": "operamc",
+    "discord": "https://discord.gg/SxZXJsRHxD",
+    "teamspeak": "operamc"
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "CityBuild",
+      "color": "#ff073a"
+    },
+    "gungame": {
+      "name": "GunGame",
+      "color": "#ff073a"
+    },
+    "freebuild": {
+      "name": "FreeBuild",
+      "color": "#ff073a"
+    }
+  },
+  "discord": {
+    "server_id": 1080577009589162005,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.